src/main/java/com/leetcode/problems/Arranging.java

```diff
@@ -0,0 +1,29 @@
+package com.leetcode.problems;
+
+// https://leetcode.com/problems/arranging-coins/
```

Comment:
Can't this be solved in O(1) time and space?

The question is, can you write n as a x(x+1)/2? If yes then all rows are complete, if not then you need to figure out the edge cases.